src/api/spec/controllers/concerns/triggerable_spec.rb

```diff
@@ -0,0 +1,173 @@
+require 'rails_helper'
+
+RSpec.describe Triggerable do
+  let(:fakes_controller) do
+    Class.new(ApplicationController) do
+      include Triggerable
+    end
+  end
+
+  let(:object) { fakes_controller.new }
```

Comment:
`fake_controller_instance` is better than `object`

Reply:
done